lib/result-view.coffee

```diff
@@ -130,14 +130,15 @@ class ResultView
                 @element.classList.add 'rich'
                 @setMultiline true
 
+            container.appendChild htmlElement
+
             if mimeType is 'text/html'
-                container.classList.add('html')
+                if @getAllText() isnt ''
+                    @element.classList.remove 'rich'
```

Comment:
Because `@getAllText()` would always return `''`.

It gets the text from the `container` see [here](https://github.com/nteract/hydrogen/blob/master/lib/result-view.coffee#L152)